kube_log_watcher/agents/scalyr.py

```diff
@@ -34,6 +34,8 @@ def __init__(self, cluster_id: str, load_template):
         self.dest_path = os.environ.get('WATCHER_SCALYR_DEST_PATH')
         self.scalyr_server = os.environ.get('WATCHER_SCALYR_SERVER')
         self.parse_lines_json = os.environ.get('WATCHER_SCALYR_PARSE_LINES_JSON', False)
+        self.cluster_alias = os.environ.get('CLUSTER_ALIAS')
+        self.cluster_environment = os.environ.get('CLUSTER_ENVIRONMENT')
```

Comment:
Can you add default values in case the environment variables are not set?

Looks good otherwise!

Reply:
done